### ISSUE TITLE
Support extensionless files

### ISSUE THREAD

**Issue by aaronmallen** [onury/docma#52]
currently there is no way to compile an extension less text document:

```json
// docma.json
{
  "src": [
    "./src/**/*.js",
    "./README.md",
    "./CHANGELOG.md",
    "./LICENSE"
  ]
}
```
will produce the following warning and fail to compile the file: 

```bash
Unsupported file ignored: /$HOME/my-project/LICENSE
```

please add support for extension less text files as this format for LICENSE is fairly common practice (its the default new repo behavior on github)

**Comment by onury** [comment]
Ok. This sounds fair.

### Notes for implementation:
We can add support for force-parsing given file(s) to a specific format. 
Format would be one of: `md`, `html`, `js`
An build configuration:
```js
{
    "src": [
        "README.md",
        "LICENSE:md",   // an extensionless file, forced to be parsed as markdown with the :md suffix
        "file.ext:html" // file with extension "ext" is forced to be parsed as HTML
    ]
}
```

I guess this resolves the problem.

**Comment by onury** [comment]
Released with [Docma v3.0.0](https://onury.io/docma/changelog).
